components/scream/src/physics/zm/CMakeLists.txt

```diff
@@ -0,0 +1,37 @@
+set(CIMEROOT ${SCREAM_BASE_DIR}/../../cime)
+list(APPEND CMAKE_MODULE_PATH ${CIMEROOT}/src/CMake)
+
+set(GENF90 ${CIMEROOT}/src/externals/genf90/genf90.pl)
+set(ENABLE_GENF90 True)
+include(genf90_utils)
+include(Sourcelist_utils)
```

Comment:
I don't see you using genf90 on any file here. Perhaps you needed it at the beginning, then as you modified zm your dependency on `*.f90.in` files went away? If so, let's remove this stuff.

Reply:
@mfossitt Should genf90 stuff be removed then? Or is it actually used?